frontend/pages/_app.js

```diff
@@ -1,26 +1,8 @@
 import '../styles/globals.css'
-import Script from 'next/script'
-import Header from '../components/Header'
-import Navbar from '../components/Navbar'
-import Footer from '../components/Footer'
 
 function MyApp ({ Component, pageProps }) {
-  return (
-    <>
-      <Script src='https://ajax.googleapis.com/ajax/libs/jquery/3.6.1/jquery.min.js' />
-      <Script
-        src='https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js'
-        integrity='sha384-ka7Sk0Gln4gmtz2MlQnikT1wXgYsOg+OMhuP+IlRH9sENBO0LRn5q+8nbTov4+1p'
-        crossorigin='anonymous'
-      />
-      <Header />
-      <Navbar />
-      <div id='app-body-content'>
-        <Component {...pageProps} />
-      </div>
-      <Footer />
-    </>
-  )
+  const getLayout = Component.getLayout || ((page) => page)
+  return getLayout(<Component {...pageProps} />)
```

Comment:
Hey Ann Marie, everything looks great, but I'm noticing that the global.css import does not seem to be included in getLayout() (I think) and so the project titles are missing the beaver orange styling.

![image](https://user-images.githubusercontent.com/25272162/218370249-f891af12-df8a-421f-9721-30abcdeeddbf.png)

Additionally, team member names are missing the styling and degree/title.
![image](https://user-images.githubusercontent.com/25272162/218370329-7cfa62df-ecda-4111-adfa-6bf547661e9f.png)

Reply:
Never mind, disregard the suggestion
I noticed that the styling was commented out on the projectcard.js file, and for the personcard.js file, it's absent